samples/subsys/usb/dfu-next/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(hid-keyboard)
```

Comment:
```suggestion
project(dfu-next)
```

Reply:
Thanks, changed to usb-dfu to not rename it again in https://github.com/zephyrproject-rtos/zephyr/pull/80985